client/verta/verta/_deployment/endpoint.py

```diff
@@ -29,10 +31,15 @@ def _create(cls, conn, conf, workspace, path, description=None):
 
     @classmethod
     def _create_json(cls, conn, workspace, path, description=None):
+        json = {}
```

Comment:
`json` is the name for a built-in Python module; I like using `data` instead to prevent name collision. Same change for the `json`s below, please!
```suggestion
        data = {}
```

Reply:
renamed